### PR TITLE
Ogc 241 map

### PR DESCRIPTION
**Pull request by BreathingFlesh**

### CODE REVIEW

File: src/onegov/core/translate_coordinates.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
```
Comment (msom) [inline src/onegov/core/translate_coordinates.py:1]: No need for encoding

File: src/onegov/org/app.py [hunk 2]
```diff
@@ -319,13 +319,15 @@ def org_content_security_policy():
     policy.child_src.add('https://*.youtube.com')
     policy.child_src.add('https://*.vimeo.com')
     policy.child_src.add('https://checkout.stripe.com')
-    policy.child_src.add('https://onegov-livechat.herokuapp.com')
+    policy.child_src.add('https://map.geo.admin.ch')
```
Comment (msom) [inline src/onegov/org/app.py:322]: Do wee need this?

File: tests/onegov/agency/test_forms.py [hunk 3]
```diff
@@ -84,7 +84,8 @@ def test_extended_agency_form(agency_app):
     }))
     data = form.get_useful_data()
     assert list(data.keys()) == [
-        'title', 'portrait', 'export_fields', 'organigram_file'
+        'title', 'portrait', 'coordinates', 'address', 'zip_code', 'city',
+        'export_fields', 'organigram_file'
     ]
```
Comment (msom) [inline tests/onegov/agency/test_forms.py:89]: Please also test actual data like the other fields

**Review by msom — CHANGES_REQUESTED** [review]

File: src/onegov/org/app.py [hunk 4]
```diff
@@ -319,13 +319,15 @@ def org_content_security_policy():
...
 
     policy.connect_src.add(SELF)
     policy.connect_src.add('https://checkout.stripe.com')
     policy.connect_src.add('https://sentry.io')
     policy.connect_src.add('https://*.google-analytics.com')
     policy.connect_src.add('https://stats.g.doubleclick.net')
+    policy.connect_src.add('https://map.geo.bs.ch')
+    policy.connect_src.add('https://wmts.geo.bs.ch')
```
[the remark this answers is not in the document]
Comment (BreathingFlesh) [reply src/onegov/org/app.py:330]: Just checked, we do.

File: src/onegov/org/app.py (hunk 2 above)
Comment (BreathingFlesh) [reply src/onegov/org/app.py:322]: We don't.

**Pull request merged**